image-loader/app/model/ImageUpload.scala

```diff
@@ -375,20 +372,22 @@ object Uploader {
   }
 
   private def createOptimisedFileFuture(uploadRequest: UploadRequest,
-                                        deps: ImageUploadOpsDependencies)(implicit ec: ExecutionContext) = {
+                                        deps: ImageUploadOpsDependencies)(implicit ec: ExecutionContext): Future[UploadRequest] = {
     import deps._
-    val uploadedFile = uploadRequest.tempFile
     uploadRequest.mimeType match {
       case Some(mime) => mime match {
         case _ if config.transcodedMimeTypes.contains(mime) =>
           for {
-            transformedImage <- imageOps.transformImage(uploadedFile, uploadRequest.mimeType, config.tempDir)
-          } yield transformedImage
+            transformedImage <- imageOps.transformImage(uploadRequest.tempFile, uploadRequest.mimeType, config.tempDir)
+          } yield (uploadRequest
+            // This file has been converted.
+            .copy(mimeType = Some(Jpeg))
```

Comment:
Do we always convert to a jpeg here? I thought under some situations we converted to `png`, but I might be thinking of @paperboyo's future plans!

Reply:
We definitely shouldn’t convert *always* to JPEG! Not even under current regime! (otherwise, we wouldn’t be able to support transparency)